crates/p2p_proto/src/propagation.rs

```diff
@@ -86,26 +88,30 @@ impl ToProtobuf<proto::propagation::Message> for Message {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, ToProtobuf, TryFromProtobuf)]
+#[cfg_attr(feature = "test-utils", derive(Dummy))]
```

Comment:
Can you link an example of where this `Dummy` thing is being used? It can be any of them; I'm somehow just missing how this is actually in use 😅

Reply:
The deal with `Dummy` is that once it's implemented, you get the implementation of `Fake` for free, which then can be used by the `Faker` (https://docs.rs/fake/2.5.0/fake/trait.Dummy.html)

https://github.com/eqlabs/pathfinder/blob/96e7b50943157d723600b022ca746b8188c4ed29/crates/p2p/src/tests.rs#L334-L351

Reply:
Thanks, and it just fills in the fields with random data basically?